doc/nrf/protocols/matter/overview/group_communication.rst

```diff
@@ -0,0 +1,228 @@
+
+#. Send a command to toggle on/off on the application endpoint (endpoint 1 includes the onoff cluster)::
+
+   ./chip-tool onoff toggle 0xffffffffffffdcba 1
+
+**LED2** on the Light Bulb should now change the state.
+
+Group Communication triggered from the accessory (Light Switch)
+===============================================================
+
+The initial settings for this use case are the following:
+
+.. code-block:: none
+
+   Light Bulb nodeid = 2
+   Light Switch nodeid = 1
+   Test group keyset
+        groupKeySetID=258
+        epochKey0=a0a1a2a3a4a5a6a7a8a9aaabacad7531
+        epochStartTime0=1110000
+        epochKey1=b0b1b2b3b4b5b6b7b8b9babbbcbd7531
+        epochStartTime1=1110001
+        epochKey2=c0c1c2c3c4c5c6c7c8c9cacbcccd7531
+        epochStartTime2=1110002
+   Test group:
+        groupId=30001
+        name=Test_Group_30001
+
+Complete the following steps to start Group Communication:
+
+1. Set the group settings on the on/off client (Light Switch)::
+
+   ./chip-tool groupkeymanagement key-set-write '{"groupKeySetID":"258","groupKeySecurityPolicy":0,"epochKey0":"a0a1a2a3a4a5a6a7a8a9aaabacad7531","epochStartTime0":"1110000","epochKey1":"b0b1b2b3b4b5b6b7b8b9babbbcbd7531","epochStartTime1":"1110001","epochKey2":"c0c1c2c3c4c5c6c7c8c9cacbcccd7531","epochStartTime2":"1110002"}' 1 0
+
+#. Set the keyset to the group map on the on/off client (Light Switch)::
+
+   ./chip-tool groupkeymanagement write group-key-map '[{"groupId":"30001","groupKeySetID":"258","fabricIndex":"1"}]' 1 0
+
+#. Set the same group settings on the on/off server (Light Bulb)::
+
+   ./chip-tool groupkeymanagement key-set-write '{"groupKeySetID":"258","groupKeySecurityPolicy":0,"epochKey0":"a0a1a2a3a4a5a6a7a8a9aaabacad7531","epochStartTime0":"1110000","epochKey1":"b0b1b2b3b4b5b6b7b8b9babbbcbd7531","epochStartTime1":"1110001","epochKey2":"c0c1c2c3c4c5c6c7c8c9cacbcccd7531","epochStartTime2":"1110002"}' 2 0
+
+#. Set the keyset to the group map on the on/off server (Light Bulb)::
+
+   ./chip-tool groupkeymanagement write group-key-map '[{"groupId":"30001","groupKeySetID":"258","fabricIndex":"1"}]' 2 0
+
+#. Add the group to the application endpoint (endpoint 1 includes the on/off cluster)::
+
+   ./chip-tool groups add-group 30001 Test_Group_30001 2 1
+
+#. Set the binding to the on/off client (Light Switch)::
+
+   ./chip-tool binding write binding '[{"fabricIndex":"1","group":"30001"}]' 1 1
+
+#. Set ACL on the on/off server (Light Bulb)::
+
+   ./chip-tool accesscontrol write acl '[{"fabricIndex":1,"privilege":5,"authMode":2,"subjects":[112233],"targets":null},{"fabricIndex":1,"privilege":3,"authMode":3,"subjects":[30001],"targets":[{"cluster":null,"endpoint":1,"deviceType":null}]}]' 2 0
+
+#. Click button **2** on the Light Switch to send the toggle message.
```

Comment:
It is BUTTON 2 for nrf53/nrf52,  for nrf54l15 is BUTTON1 (because numeration is different on DK)

Reply:
Done.